### PR TITLE
feat(sweeps): basic docs for Optuna scheduler

### PR DESCRIPTION
**Pull request by gtarpenning**
## Description

Adds information about the optuna scheduler

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/launch/sweeps-on-launch.md [hunk 1]
```diff
@@ -171,6 +172,56 @@ scheduler:
 ...
 ```
 
+  </TabItem>
+  <TabItem value="optuna-scheduler">
+
+  Create a launch sweep using Optuna's scheduling logic with a Job. Either create your own, steps in the [`wandb/launch-jobs`](https://github.com/wandb/launch-jobs/jobs/sweep_schedulers/optuna_scheduler) repo, or use the pre-built `Optuna Scheduler Image Job` in the `wandb/jobs` project. Full job name: `'wandb/jobs/Optuna Sweep Scheduler:latest'`.
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md]: QQ on this: "Either create your own, steps in the [`wandb/launch-jobs`](https://github.com/wandb/launch-jobs/jobs/sweep_schedulers/optuna_scheduler) repo, or use the pre-built `Optuna Scheduler Image Job` in the `wandb/jobs` project." 

Would it make sense to rewrite this as:

You can either create your own job or use a pre-built Optuna shedular image job. 

See the [`wandb/launch-jobs`](https://github.com/wandb/launch-jobs/jobs/sweep_schedulers/optuna_scheduler) repo for examples on how to create your own job. Pull the `'wandb/jobs/Optuna Sweep Scheduler:latest'` job from `wandb/jobs` project to use the pre-built  Optuna shedular image job.

File: docs/guides/launch/sweeps-on-launch.md [hunk 2]
```diff
@@ -171,6 +172,56 @@ scheduler:
 ...
 ```
 
+  </TabItem>
+  <TabItem value="optuna-scheduler">
+
+  Create a launch sweep using Optuna's scheduling logic with a Job. Either create your own, steps in the [`wandb/launch-jobs`](https://github.com/wandb/launch-jobs/jobs/sweep_schedulers/optuna_scheduler) repo, or use the pre-built `Optuna Scheduler Image Job` in the `wandb/jobs` project. Full job name: `'wandb/jobs/Optuna Sweep Scheduler:latest'`. 
+
+  Now, we can create a sweep configuration that uses this job. Include a `scheduler` block that with a `job` key pointing to the Optuna scheduler job (example below).
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md]: "Now, we can create a sweep configuration that uses this job...." -> "After you create a job, you can now create a sweep...."

File: docs/guides/launch/sweeps-on-launch.md [hunk 3]
```diff
@@ -171,6 +172,56 @@ scheduler:
...
+
+  Finally, we can launch the sweep to a an active queue with the launch-sweep command:
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md]: Finally, we can launch the sweep to a an active queue with the launch-sweep command -> Lastly, launch the sweep to a an active queue with the launch-sweep command:

File: docs/guides/launch/sweeps-on-launch.md [hunk 4]
```diff
@@ -171,6 +172,56 @@ scheduler:
...
+
+  parameters:
+    learning_rate:
+      min: 0.0001
+      max: 0.1
+  ```
+
+  ### Why use Optuna? 
+  
+  Optuna is a hyperparameter optimization framework that uses a variety of algorithms to find the best hyperparameters for a given model (similar to Wandb). In addition to the [sampling algorithms](https://optuna.readthedocs.io/en/stable/reference/samplers/index.html), Optuna also provides a variety of [pruning algorithms](https://optuna.readthedocs.io/en/stable/reference/pruners.html) that can be used to terminate poorly performing runs early. This is especially useful when running a large number of runs, as it can save time and resources. The classes are highly configurable, just pass in the expected parameters in the `scheduler.settings.pruner/sampler.args` block of the config file.
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md]: Wandb -> W&B

**Review by ngrayluna — CHANGES_REQUESTED** [review]
Nice. No major stuff, just minor word smithing.

**Review by ngrayluna — APPROVED** [review]

**Comment by ngrayluna** [comment]
Looks good to me. Is it safe to merge?

**Comment by gtarpenning** [comment]
Yep, those jobs have now had their guts relocated, rebuilding and will merge this when the jobs are out of the ICU and on their feet. Probably tn @ngrayluna
